rpc/legacyrpc/methods.go

```diff
@@ -1384,14 +1385,36 @@ func LockUnspent(icmd interface{}, w *wallet.Wallet) (interface{}, error) {
 	return true, nil
 }
 
+func makeOutputs(pairs map[string]btcutil.Amount, chainParams *chaincfg.Params) ([]*wire.TxOut, error) {
```

Comment:
Could probably use a comment here.  It's not horribly hard to figure out the intention, but it might be useful to document the outputs are created in random order.